telosnetwork/teloscan
# Adding new option to the header menu to change the language

# Fixes #302

## Description
This PR includes a new option in the header menu to change the current language.

## Blocked 
This PR is blocked until merging the [incorporation of the library i18n PR](https://github.com/telosnetwork/teloscan/pull/292) and at least one of the following language incorporations:
- https://github.com/telosnetwork/teloscan/pull/301

## Test scenarios
- Open the header menu
- open the Language sub-menu
- You should see
   - one entry for each language present
   - the current language should be selected
- change the current selection
- the language should change and the page should reload immediately showing the corresponding Language

## Checklist:
